angular.json

```diff
@@ -50,7 +50,20 @@
               }
             ],
             "scripts": ["./node_modules/intro.js/intro.js"],
-            "allowedCommonJsDependencies": ["lottie-web", "firebase/app", "@firebase/auth"]
+            "allowedCommonJsDependencies": [
```

Comment:
Changes here are just to supress warnings in the console about 3rd party libraries that use different build systems to angular